package.json

```diff
@@ -23,7 +23,7 @@
   },
   "scripts": {
     "setup": "node ./hosting/scripts/setup.js && yarn && yarn bootstrap && yarn build && yarn dev",
-    "bootstrap": "lerna bootstrap",
+    "bootstrap": "lerna bootstrap && cd packages/pro && lerna bootstrap && yarn setup",
```

Comment:
I think the bootstrap shouldn't run the setup, is that an infinite loop?

Reply:
It runs the bootstrap in pro. For the time being is still required because pro uses lerna as well. This will be removed when the pro repo gets rid of the lerna dependency to become a single package